src/config.js

```diff
@@ -1,5 +1,5 @@
 /* TODO: fill this in!*/
-const CONTRACT_NAME = process.env.CONTRACT_NAME || "new-corgis";
+const CONTRACT_NAME = process.env.anaiscorgis || "new-corgis";
```

Comment:
this is kind of weird. Why do you change this?

Reply:
Upps that was not on purpose :)